bc-wikitemplate.md

```diff
@@ -1,87 +1,103 @@
-  - [ ] bookmarks in the bookmark folder toolbar can be opened
+  - [ ] stored passwords are preserved
+  - [ ] unpinned tabs can be pinned
 
-## About pages
+### About pages
 
 - [ ] Test that brave:// forwards to chrome://
-- [ ] chrome://adblock
-- [ ] chrome://payments
-- [ ] chrome://newtab
-- [ ] chrome://welcome
-- [ ] chrome://version
-
-## Import
-
-- [ ] Check that you can import from browser-laptop
-- [ ] Check that you can import from Google Chrome (We implemented all ourselves)
-- [ ] Check that you can import from Firefox (Cookies is all we implemented ourselves)
-- [ ] Check that you can import from Edge (No custom code we implemented ourselves)
+- [ ] Test chrome://adblock loads adblock page
+- [ ] Test chrome://rewards loads Brave rewards page 
+- [ ] Test chrome://newtab loads a new tab
+- [ ] Test chrome://welcome loads the welcome page 
+- [ ] Test chrome://version correctly shows Brave version and Chromium version 
 
-## Context menus
+### Import
 
-- [ ] Block this element option
+- [ ] Check that you can import bookmarks, cookies, history, passwords and stats from Brave(browser-laptop)
+- [ ] Check that you can import bookmarks, cookies, history, passwords from Google Chrome 
+- [ ] Check that you can import bookmarks, cookies, history, passwords, autofill and search engines from Firefox
+- [ ] Check that you can import bookmarks from Edge
 
-## Keyboard Shortcuts
+### Context menus
 
-- [ ] TODO: Custom ones we define for functionality not in Chrome but in brave-core
+- [ ] Verify `Block element via selector` removes a CSS element from page without reloading
+- [ ] Verify `Clear all CSS rules for this sites` removes the blocked element after page reload
+- [ ] Verify `Clear all CSS rules for all sites` removes the rule and loads all elements for all pages
 
-## Printing PDF
-
-- [ ] Test that you can print a PDF
 
 ## Extensions/Plugins tests
 
-- [ ] At least one thing on Brave Update server is installable (Example: Ad-block DAT file on fresh extension)
-- [ ] At least one thing on Google Update server is installable
-- [ ] PDFJS extension gets instaslled automatically on fresh profile
+- [ ] Verify one item from Brave Update server is installable (Example: Ad-block DAT file on fresh extension)
+- [ ] Verify one item from Google Update server is installable(Example: Extensions from CWS)
+- [ ] Verify PDFJS, Torrent viewer extensions are installed automatically on fresh profile and cannot be disabled
 - [ ] Test that an old extension gets updated to a new one from an old profile on Google server
 - [ ] Test that an old extension gets updated to a new one from an old profile on Brave server
+- [ ] Verify magnet links loads Torrent viewer page and able to download torrent
 
-### Widevine
-- [ ] Test that you can log into Netflix and start a show from a fresh profile
+### CWS
+
+- [ ] Verify installing uBO/ABP from CWS shows warning message `NOT A RECOMMENDED BRAVE EXTESION!`but still gets installed
```

Comment:
I think you can just pick a single extension. Doesn't have to be uBlock or ABP either.